package.json

```diff
@@ -61,7 +61,6 @@
     "html-webpack-plugin": "^2.24.1",
     "mocha": "^3.0.2",
     "node-sass": "^3.8.0",
-    "postcss-loader": "^1.2.0",
```

Comment:
because i meet same problem, like this.
https://github.com/postcss/postcss-loader/issues/128

so I decide that I add it back  when I really need that.